cmd/test/cmd.go

```diff
@@ -70,10 +70,8 @@ type builder interface {
 func Test(ctx context.Context, ioCtrl *io.Controller, k8sLogger *io.K8sLogger, at *analytics.Tracker) *cobra.Command {
 	options := &Options{}
 	cmd := &cobra.Command{
-		Use:          "test",
-		Short:        "Run tests",
-		Hidden:       true,
-		SilenceUsage: true,
+		Use:   "test",
+		Short: "Run tests defined in your okteto manifest",
```

Comment:
why don't we also link to docs? (ie. https://www.okteto.com/docs/reference/okteto-cli/#test)